### ISSUE TITLE
[Discussion] : Ideas for filters layout

### ISSUE THREAD

**Issue by joaogarin** [contentacms/contenta_angular#25]
I was thinking about some approaches to theming the filters. Some of these approaches can work better the more filters you start adding, I really like the approach right now but the amount of filters (if one starts adding more filters e.g. date with a nice [datepicker](https://material.angular.io/components/datepicker/overview) and so on) can make the page look a bit bloated..

I basically narrowed it down to a couple options I think we could go with. I think right now it works but in terms of layout it can be much better..but in any case might be worth exploring some possible choices for this.

1 ) Sidenav with filters. Basically this would be like a toggle-able sidenav that would open the filters "side panel". We then could display the filters in a nice manner maybe even with some nice descriptions of what some filters actually mean.

2 ) A [Dialog](https://material.angular.io/components/dialog/overview) window with the filters panel. This would basically open a Dialog with the filter configuration panel which could make for less noise on the actual list also.

3 ) A sort of facet approach with the filters displayed each one as a block or something simillar (quite common in search heavy content sites also)

4 ) the current approach with the filters displayed always in the list.  I think this approach while working ok can become unbearable if the filters list grows long and its always there occupying some room when you actually already set the filters (but I like the idea that is visible what the current search is)

**Comment by mrjmd** [comment]
I agree that the current approach is not going to look good as we add more filters (categories, pager, etc). It already takes up too much real estate on my mobile.

A sidenav or faceted approach could work, I'm not a fan of the dialog approach though.

I also noticed there's a small bug with the filters as I built them, where the placeholder text for the Prep Time and Total filters sticks around after you've made a selection. Haven't had time to look into it yet.
